src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -522,6 +522,10 @@ std::wstring Terminal::GetHyperlinkAtBufferPosition(const til::point bufferPos)
     // Case 2 - Step 2: get the auto-detected hyperlink
     if (result.has_value() && result->value == _hyperlinkPatternId)
     {
+        // GetPlainText works with inclusive coordinates, but interval's stop
+        // point is (horizontally) exclusive, so let's just update it.
+        result->stop.x--;

```

Comment:
Note to other reviewers: This works because of line 1425 in `Terminal::_getPatterns`.